src/Utils/UtilsClient.ts

```
@@ -0,0 +1,13 @@
+const BASE_URL = 'http://localhost:5173/public/api';
+
+export function getData<T>(url: string): Promise<T> {
+  return fetch(BASE_URL + url).then(response => {
```

Concatenating `BASE_URL + url` can produce incorrect URLs (double slashes) or silently accept absolute `url` arguments. Prefer constructing the final URL robustly, e.g. `new URL(url, BASE_URL).toString()` or normalizing slashes, and validate that callers pass only expected endpoint paths.